docs/packaging/package.yml.md

```diff
@@ -3,15 +3,15 @@ title: Package YAML
 summary: Learning the package.yml packaging format
 ---
 
-# Package.yml
+# `package.yml`
 
-All packages are generated from a single build file, which provides all of the required metadata for the package manager, plus the packaging steps involved to produce a package. This follows the YAML specification.
+Each package is generated from a single build file named `package.yml`. This provides all of the required metadata for the package manager, plus the packaging steps involved to produce a package. This file follows the YAML specification.
```

Comment:
Consider explicitly using the word 'recipe' here rather than 'file'? This meshes well with the use of "packaging steps".

Reply:
I disagree with this. The word "file" exactly describes the thing being referenced (the package.yml file). The word recipe is ambiguous and would be confusing.

I've personally only ever seen a yaml file called a recipe in the context of chef software, and some previous conversations in chat among Solus staff. I think it would be confusing to new packagers without adding value. I'm willing to listen to what Evan and other reviewers have to say.

Reply:
I came up with wording that I think is a good compromise, which introduces the concept of this file being a recipe.